Landertron.netkan

```diff
@@ -0,0 +1,15 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "Landertron",
+    "$kref"        : "#/ckan/github/Kerbas-ad-astra/XTLandertron",
+    "recommends": [
+        { "name": "HGR" }
+    ],
+    "license"      : "GPL-3.0",
+    "name"         : "Landertrons",
+    "abstract"     : "Automatic retro-rockets for landing.",
+    "ksp_version"  : "1.2.99",
```

Comment:
Ah my bad, I just checked what other mods had in their netkan file! Feel free to change it!